src/Observable.ts

```diff
@@ -96,9 +97,10 @@ export class Observable<T> implements CoreOperators<T>  {
       }
     } else {
       const next = <((x?) => void)> observerOrNext;
-      subscriber = Subscriber.create(next, error, complete);
+      subscriber = Subscriber.create(next, error, complete, start);
     }
 
+    subscriber.start(subscriber);
     subscriber.add(this._subscribe(subscriber));
```

Comment:
@blesh yes, but if the `start` method synchronously calls `unsubscribe` (thus setting `isUnsubscribed` to `true`), then there's no reason to execute line 104 (`subscriber.add(this._subscribe(subscriber));`).